src/lib/jsonParseV2/parseEntity.cpp

```diff
@@ -79,17 +79,18 @@ std::string parseEntity(ConnectionInfo* ciP, Entity* eP, bool eidInURL)
     {
       LM_W(("Bad Input (No entity id specified"));
       eP->errorCode.fill("ParseError", "no entity id specified");
-      ciP->httpStatusCode = SccBadRequest;;
+      ciP->httpStatusCode = SccBadRequest;
+
       return eP->render(ciP, EntitiesRequest);
     }
   }
-  else if (document.Empty()) 
+  else if (document.ObjectEmpty()) 
   {
     LM_W(("Bad Input (Empty payload)"));
     eP->errorCode.fill("ParseError", "empty payload");
-    ciP->httpStatusCode = SccBadRequest;;
-    return eP->render(ciP, EntitiesRequest);
+    ciP->httpStatusCode = SccBadRequest;
```

Comment:
Just style changes, yes.
For readability, it is good to have an empty line after the variable declarations and before the return statement.
This is my opinion, of course. But also, most of the source code of the broker is written this way, so it could be called to fix the style as well.

I mention this in the style guide.
Empty lines DO help readability, I don't think anyone can argue that.
They just need to be in the right place.